wurst/data/LinkedList.wurst

```diff
@@ -244,7 +246,7 @@ public class LLStaticIterator<T>
 			current.prev.next = current.next
 			current.next.prev = current.prev
 			let removedNode   = current
-			current           = current.prev
+			current		   = current.prev
```

Comment:
I see that you fixed mixed whitespace -> tabs only. Much prefer spaces only for visual alignment.